specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/preview/2020-11-01-preview3/phonenumbers.json

```diff
@@ -0,0 +1,1014 @@
+      "description": "",
+      "type": "string",
+      "enum": [
+        "person",
+        "application"
+      ],
+      "x-ms-enum": {
+        "name": "AssignmentType",
+        "modelAsString": true
+      }
+    },
+    "MonthlyRate": {
+      "description": "Represents a wrapper of rate information",
+      "type": "object",
+      "properties": {
+        "value": {
+          "format": "double",
+          "description": "The monthly rate of a phone plan group",
+          "type": "number"
+        },
+        "currency": {
+          "description": "The currency of a phone plan group",
+          "enum": [
+            "USD"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "Currency",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "SearchRequest": {
+      "description": "",
+      "required": [
+        "numberType",
+        "assignmentType",
+        "country",
+        "capabilities"
+      ],
+      "type": "object",
+      "properties": {
+        "numberType": {
+          "description": "",
+          "$ref": "#/definitions/PhoneNumberType"
+        },
+        "assignmentType": {
+          "description": "",
+          "$ref": "#/definitions/AssignmentType"
+        },
+        "capabilities": {
+          "description": "",
+          "$ref": "#/definitions/Capabilities"
+        },
+        "areaCode": {
+          "description": "The area code of the search.",
+          "type": "string"
+        },
+        "quantity": {
+          "format": "int32",
+          "description": "The quantity of phone numbers in the search.",
+          "type": "integer"
+        }
+      }
+    },
+    "SearchResponse": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "",
+          "type": "string"
+        },
+        "phoneNumbers": {
+          "description": "",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "numberType": {
+          "description": "",
+          "$ref": "#/definitions/PhoneNumberType"
+        },
+        "assignmentType": {
+          "description": "",
+          "$ref": "#/definitions/AssignmentType"
+        },
+        "capabilities": {
+          "description": "",
+          "$ref": "#/definitions/Capabilities"
+        },
+        "monthlyRate": {
+          "description": "",
+          "$ref": "#/definitions/MonthlyRate"
+        },
+        "searchExpiresBy": {
+          "format": "date-time",
+          "description": "The date that search expires and the numbers become available.",
+          "type": "string"
+        }
+      }
+    },
+    "CountryCapability": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "numberType": {
+          "description": "",
+          "$ref": "#/definitions/PhoneNumberType"
+        },
+        "assignableTo": {
```

Comment:
Just want to clarify, if a country capabilities is available for both User and Application, they will appear as two separate country capabilities?

Reply:
Yes, that was the idea. Only capabilities get grouped into an array, all other differences in property values like numberType or assignmentType would show up as a new item in the response array.